lib/rucio/core/replica.py

```diff
@@ -1978,11 +2041,13 @@ def __cleanup_after_replica_deletion(scope_name_temp_table, scope_name_temp_tabl
     # delete empty dids
     if did_condition:
         for chunk in chunks(did_condition, 10):
-            query = session.query(models.DataIdentifier.scope,
-                                  models.DataIdentifier.name,
-                                  models.DataIdentifier.did_type). \
-                with_hint(models.DataIdentifier, "INDEX(DIDS DIDS_PK)", 'oracle'). \
-                filter(or_(*chunk))
+            stmt = select(
+                models.DataIdentifier.scope,
+                models.DataIdentifier.name,
+                models.DataIdentifier.did_type,
+            ).with_hint(models.DataIdentifier, "INDEX(DIDS DIDS_PK)", 'oracle').\
+                where(or_(*chunk))
```

Comment:
fix the formatting

Reply:
Formatting should be following new conventions in the new commit now